### PR TITLE
solved labels to sentence case

### LINKED ISSUE #7799: Change labels to sentence case for Form component
### Problem

There is an inconsistency in the text casing, some labels under `Properties` and `Styles` tab in the component manager are not in [Sentence case](https://www.k-state.edu/grad/academics/etdr/write/sentence-case.html).

### What is the expected behaviour?

All the labels under the `Properties` and `Styles` tab in the component managers should be in the Sentence case.

### What is the current behaviour?

The below labels are not in Sentence case(changes are only needed in the below labels).

| Current Values       | Expected Values       |
| :------------------- | :-------------------- |
| Button To Submit Form        | Button to submit form         |
| Loading State | Loading state  |
| Background Color | Background color |
| Border Radius        | Border radius         |

### How to reproduce the issue?

1. Open ToolJet.
2. Create a new app.
3. Search for the component and drag it onto the Canvas.
4. Click on the component to open the Component Manager in the right sidebar.
5. Examine the labels under both the `Properties` and `Styles` tabs. Some labels are not in sentence case and should be updated accordingly.

### Additional Information

1. Changes are only required in the `widgetConfig.js` file, locate the object which has the related component name in the name key, and make changes only in that particular object.  
**For e.g. for table component, you will make changes in the below object of the widgetConfig.js file:**
```
{
    name: 'Table',
    displayName: 'Table',
    description: 'Display paginated tabular data',
    component: 'Table',
.....
}
```
2. Do not make changes to any other labels, files or components.

### Screenshots or Screencast

**Example** using the table component, items that are not in Sentence case are highlighted in red, these are inconsistent with other labels which are in Sentence case

image

### PR DESCRIPTION
**Pull request by ronisarkarexe**
Issue
closed #7799

Solved inconsistency in the text casing, changed labels to sentence case for Form component

### CODE REVIEW

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @ronisarkarexe, I request you to revisit the issue description https://github.com/ToolJet/ToolJet/issues/7799 and be sure to update the PR with changes that are only requested in the table. 

Kindly revert all the other changes which are not relevant to the form component and which are not mentioned in the issue. Kindly update the PR to progress it for further reviews.

**Comment by tejakummarikuntla** [comment]
Hey @ronisarkarexe, were you able to look into the suggested changes?

**Review by withshubh — CHANGES_REQUESTED** [review]
You have to make changes only for the form component.
Search for `component: 'Form',` in the file
or go to line number 5672

**Comment by ronisarkarexe** [comment]
> Hey @ronisarkarexe, were you able to look into the suggested changes?

Could you please verify if this is correct?

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -2295,7 +2295,7 @@ export const widgets = [
     properties: {
       loadingState: {
         type: 'toggle',
-        displayName: 'loading state',
+        displayName: 'Loading State',
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:2298]: @ronisarkarexe, please revert this change.

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @ronisarkarexe, kindly address the suggested changes.

**Comment by withshubh** [comment]
Closing this pull request due to inactivity.

**Pull request closed without merging**